### ISSUE TITLE
actions routing docs

### ISSUE THREAD

**Issue by ra0o0f** [arangodb/arangodb#1880]
Hi

the routing match example at the bottom of [HtmlExample](https://docs.arangodb.com/ArangoActions/HtmlExample.html), does `/hello/you` match with route `4` or `2` ?

thanks

```
(1) { url: { match: "/hello/world" } }
(2) { url: { match: "/hello/:name", constraint: { name: "/[a-z]+/" } } }
(3) { url: { match: "/:something/world" } }
(4) { url: { match: "/hello/*" } }
Then

/hello/world is match by (1)
/hello/emil is match by (2)
/your/world is match by (3)
/hello/you is match by (4)
```

**Comment by pluma** [comment]
Hi ra0o0f, could you explain what you are trying to do? These days we generally recommend using Foxx services if you want to extend the HTTP API with custom endpoints instead of messing with the actions API directly. The actions and collection-based modules are mostly a holdover from the pre-Foxx days.

ArangoDB 3.0 aims to make it considerably easier to get up and going with Foxx. You can check out the latest pre-release for ArangoDB 3 at https://www.arangodb.com/download-preview/ or the final release later this week. The in-progress version of the updated Foxx documentation can always be found at https://docs.arangodb.com/devel/Manual/Foxx/ and the final 3.0 docs will be available at https://docs.arangodb.com/3.0/Manual/Foxx/ .

**Comment by dothebart** [comment]
@ra0o0f yes, you're right, `you` will be caught by (2). Sounds like a ZBus Error by the original Documentation Author ;-) I will fix the examples, and make them executable, as we do it with most of the other examples to ensure correctness.

Thanks for spotting this.

**Comment by dothebart** [comment]
fixed with 05ad6cface7f46dba3135aff74799257abe16e64 - https://github.com/arangodb/arangodb/blob/devel/Documentation/Books/Manual/Appendix/Deprecated/Actions/HtmlExample.mdpp 

Thanks for reporting!
